renovate.json

```diff
@@ -1,5 +1,5 @@
 {
     "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-    "schedule": "before 8am on Tuesday",
+    "schedule": "* 8 1 * *",
```

Comment:
```suggestion
    "schedule": "* 0-8 1 * *",
```

[The documentation](https://docs.renovatebot.com/configuration-options/#schedule:~:text=Similarly%2C%20if%20you,frequently%20than%20that.) recommends to use a range of several hours.


> Similarly, if you set too restrictive of a Renovate schedule and the chance of Renovate running on your repo during those hours is low, then you might find your dependency updates regularly skipped. For this reason we recommend you allow a time window of at least 3-4 hours in any schedule, unless your instance of Renovate is expected to run more frequently than that.